### PR TITLE
Allow correct spacing between consecutive fieldset

### LINKED ISSUE #1081: Consecutive fieldsets do not have adequate bottom spacing
screen shot 2017-05-19 at 17 28 43

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Allow correct spacing between consecutive fieldset

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/base/forms/fieldset/
- Edit the example html file and copy/paste the fieldset so there are two in the page
- Verify that the space between the two consecutive fieldsets is `1.5rem`

## Details

Fixes #1081

### CODE REVIEW

File: scss/_base_forms.scss [hunk 1]
```diff
@@ -134,6 +134,10 @@
       padding: .9375rem 1.25rem;
     }
 
+    & + & {
+      margin-top: $sp-large;
```
Comment (anthonydillon) [inline scss/_base_forms.scss]: Shouldn't you use `default-vertical-spacing` mixin?

**Comment by anthonydillon** [comment]
Damn reset setting the margin-top to 0.

**Comment by barrymcgee** [comment]
@anthonydillon Forgot that was a mixin - now fixed..

**Comment by barrymcgee** [comment]
@yaili http://vanilla-framework-1081-consecutive-fieldsets.demo.haus/vanilla-framework/examples/base/forms/fieldset/

To demo, copy and paste the fieldset in Inspector to view two consecutive fieldsets.

**Comment by yaili** [comment]
Can we remove the 2px right/left margin from the fieldset?

Can we keep the [current](https://www.ubuntu.com/cloud/contact-us?product=contextual-footer-cloud) spacing, 24px? 32px looks like they're disjointed.

@spencerbygraves thoughts?

**Comment by spencerbygraves** [comment]
Perhaps the gap looks too big as the fieldset has no title. Should there always be a title?

**Comment by yaili** [comment]
@spencerbygraves it doesn't always have a title, I think. The title would be inside the fieldset.

**Comment by spencerbygraves** [comment]
@yaili I don't mind the spacing

**Pull request merged**